dashboard/app/controllers/activities_controller.rb

```diff
@@ -208,15 +208,15 @@ def trophy_check(user)
       pct = counts[:current].to_f/counts[:max]
 
       new_trophy = Trophy.find_by_id case
-        when pct == Trophy::GOLD_THRESHOLD
-          Trophy::GOLD
-        when pct >= Trophy::SILVER_THRESHOLD
-          Trophy::SILVER
-        when pct >= Trophy::BRONZE_THRESHOLD
-          Trophy::BRONZE
-        else
-          # "no trophy earned"
-      end
+                                     when pct == Trophy::GOLD_THRESHOLD
+                                       Trophy::GOLD
+                                     when pct >= Trophy::SILVER_THRESHOLD
+                                       Trophy::SILVER
+                                     when pct >= Trophy::BRONZE_THRESHOLD
+                                       Trophy::BRONZE
+                                     else
+                                       # "no trophy earned"
+                                     end
```

Comment:
To avoid the extra indentation, we could set the following in `.rubocop.yml`:
```yml
Lint/EndAlignment:
  AlignWith: variable
```
This might align better with how we've been writing these statements in Dashboard up until now.